packages/orbit-components/src/Drawer/__tests__/index.test.tsx

```diff
@@ -16,17 +16,37 @@ describe("Drawer", () => {
     expect(screen.getByTestId("test")).toBeInTheDocument();
     expect(screen.getByText("content"));
   });
+
   it("should be able to toggle visibility", () => {
-    const { rerender } = render(<Drawer shown={false}>content</Drawer>);
-    expect(screen.getByText("content")).not.toBeVisible();
-    rerender(<Drawer shown>content</Drawer>);
-    expect(screen.getByText("content")).toBeVisible();
-    rerender(<Drawer shown={false}>content</Drawer>);
+    const { rerender } = render(
+      <Drawer shown={false} dataTest="test">
+        content
+      </Drawer>,
+    );
+
+    expect(screen.getByTestId("test")).toHaveAttribute("aria-hidden", "true");
```

Comment:
change to attribute matcher, as testing-library matches `toBeVisible` was not working properly as perhaps [criteria were not matched](https://github.com/testing-library/jest-dom#tobevisible) :shrug: But asserting it with attributes is also ok imo.